presto-parser/src/main/java/io/prestosql/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -707,4 +707,12 @@ protected Void visitLateral(Lateral node, C context)
 
         return null;
     }
+
+    @Override
+    protected Void visitLambdaExpression(LambdaExpression node, C context)
```

Comment:
Seems it would be worthwhile to have a test ensureing DTV implements everything from the AstVisitor interface.
https://github.com/prestosql/presto/blob/c74b775e103a7a239aafef1b7ca15fe6370495fe/presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestAllowAllAccessControl.java#L26 could be a good starting point

Reply:
Agreed, but out of scope for this PR

Reply:
> Agreed

Cool. Issue ?

Reply:
https://github.com/prestosql/presto/issues/5694